puppet/zulip/manifests/tornado_sharding.pp

```diff
@@ -26,6 +26,14 @@
     replace => false,
   }
 
+  exec { 'write_updated_sharding':
+    command   => "${::zulip_scripts_path}/lib/sharding.py",
+    unless    => "${::zulip_scripts_path}/lib/sharding.py --verify",
```

Comment:
Is `unless` the right version of this?

Reply:
Per [the docs](https://puppet.com/docs/puppet/5.5/types/exec.html#exec-attribute-unless):
> **unless**
> A test command that checks the state of the target system and restricts when the exec can run. If present, Puppet runs this test command first, then runs the main command unless the test has an exit code of 0 (success).

Reply:
OK, maybe worth adding a comment since that's not what I'd expect on a basic reading...